charts/jaeger/templates/allinone-deploy.yaml

```diff
@@ -105,6 +124,10 @@ spec:
         {{- toYaml . | nindent 12 }}
       {{- end }}
           volumeMounts:
+          {{- if .Values.v2.enabled }}
```

Comment:
I guess we should assume that v2 is always enabled and the only one we support from now on in the main branch. Then there can be v1 branch respectively.